src/System.IO/tests/TextReader/TextReaderImpl.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO.Tests
+{
+    public class TextReaderImpl : TextReader
```

Comment:
Rather than `TextReaderImpl`, it'd be better to be descriptive about what this is, e.g. `CharArrayTextReader`, or something like that.

Reply:
Changed to CharArrayTextReader...